content-blocker-lib-ios/src/ContentBlocker.swift

```diff
@@ -246,7 +231,7 @@ extension ContentBlocker {
                     var str = jsonString
```

Comment:
The str variable is initialized modified but doesn't used anymore, maybe we need to remove it. If I am missed anything please correct me.

Reply:
Sorry, as this PR is already merged. We can remove this lines if needed during other ticket. :)